src/query.js

```diff
@@ -17,7 +17,7 @@
 // Also, users name variables and want the same name back when stuff is printed
 /* jsl:option explicit*/ // Turn on JavaScriptLint variable declaration checking
 
-const IndexedFormula = require('./indexed-formula')
+const IndexedFormula = require('./store')
```

Comment:
I find this renaming confusing. Is it a store, an indexed formula, both, or are both the same? Seems that they are the same, but would use one term.

Reply:
Both  "IndexedFormula" is a formula which has indexes and be queries.   A formula in N3 is a thing in the grammar which you can put in a n3 file or between two { }.  I think devs find that nomenclature .  If we change the names of these classes then we have to leave the old ones in forever more or less.  But that's ok.  I don't know how Store works with the RDFJS folks.   I think Store is just much the most logical for devs.  So we should add Store as === IndexedFormula and move toward it.

Reply:
It makes sense to me, but that is given my N3 background. At the moment, the mixed import is confusing:

```
const IndexedFormula = require('./store')
```

I would expect either one of these:
```
const Store = require('./store')
const IndexedFormula = require('./indexed-formula')
```

> If we change the names of these classes then we have to leave the old ones in forever more or less.

The name of the `const` is up to the one doing the require.

Reply:
We need people to be able to say 
```
const kb = new $rdf.Store()
or
const kb = $rdf.IndexedFormula()
```
is what I meant

Reply:
Just changing this in the exports should be fine then.